packages/search/src/snippet-index.ts

```diff
@@ -47,22 +42,63 @@ ORDER BY
   score DESC
 LIMIT ?`;
 
+export enum SnippetType {
+  FileChunk = 'file-chunk',
+}
+
+export type SnippetId = {
```

Comment:
Adding structure to the SnippetId makes it easier for clients to interpret the different types of search results that they may get.